ingest/importer/importer.py

```diff
@@ -61,6 +62,7 @@ def import_file(self, file_path, submission_url, is_update=False, project_uuid=N
             template_mgr = None
             spreadsheet_json, template_mgr, errors = self.generate_json(file_path, is_update, project_uuid=project_uuid)
             entity_map = EntityMap.load(spreadsheet_json)
+            self.ingest_api.delete_submission_errors(submission_url)
```

Comment:
Well, I am not sure this call belongs to here. Shall we move it to the broker?

Reply:
why broker?

Reply:
The method report_errors is being called in this class, if I'm going to move the delete_submission_errors to broker, we should also move that.

Reply:
Let's leave it like this for now as time pressure, but would be nice to refactor this later as this method is doing too much.